pages/linux/systemd-analyze.md

```diff
@@ -9,3 +9,11 @@
 - Print a tree of the time critical chain of units:
 
 `systemd-analyze critical-chain`
+
+- Create a SVG file showing when each system service started, highlighting the time they spent on initialization:
+
+`systemd-analyze plot > bootup.svg`
```

Comment:
Why `bootup`, here? Would more generic work?
```suggestion
`systemd-analyze plot > {{path/to/file.svg}}`
```

Reply:
Same with `{{dependencies.svg}}` below

Reply:
Fine by me. I just thought giving a predefined file name might help you understand what the command does. Opinion?

Reply:
i like the {{bootup}} / {{dependency}} format for readability, but as @marchersimon says, this is how it's done (path/to/file). @bl-ue You aggreed with my idea, anything else to add?

Reply:
Personally, I think the decription makes clear what the file is for and using `path/to/file.svg` shows that this can be any arbitrary file. When needed for clarification we can still use `path/to/bootup_file.svg`.